jmxtrans-output/jmxtrans-output-core/src/test/java/com/googlecode/jmxtrans/model/output/OpenTSDBGenericWriterTests.java

```diff
@@ -82,7 +85,7 @@ public void setupTest() {
 		Mockito.when(this.mockResult.getClassName()).thenReturn("X-DOMAIN.PKG.CLASS-X");
 		Mockito.when(this.mockResult.getTypeName()).
 				thenReturn("Type=x-type-x,Group=x-group-x,Other=x-other-x,Name=x-name-x");
-
+		Mockito.when(this.mockServer.getLabel()).thenReturn("myhostname");
```

Comment:
It is probably not necessary to use a mock here. One of the fixture in `ServerFixtures` should work just as well.